src/msg/async/AsyncConnection.cc

```diff
@@ -417,7 +417,7 @@ void AsyncConnection::process()
       case STATE_OPEN_TAG_ACK:
         {
           ceph_le64 *seq;
-          r = read_until(sizeof(seq), state_buffer);
+          r = read_until(sizeof(*seq), state_buffer);
```

Comment:
... I'm not sure... sizeof(seq) == sizeof(*seq) in x86-64 arch. in current ceph supported archs, it should be no effect.